CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h

```diff
@@ -291,6 +294,18 @@ extern void _cf_uuid_unparse(const _cf_uuid_t uu, _cf_uuid_string_t out);
 extern void _cf_uuid_unparse_lower(const _cf_uuid_t uu, _cf_uuid_string_t out);
 extern void _cf_uuid_unparse_upper(const _cf_uuid_t uu, _cf_uuid_string_t out);
 
+#if !__COREFOUNDATION_FORFOUNDATIONONLY__
```

Comment:
Removing the guard results in the following warning:

CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h:298:45: Redefinition of typedef 'CFKeyedArchiverUIDRef' is a C11 feature

as ForFoundationOnly.h is included when building CoreFoundation.